assets/js/modules/globalNav.js

```diff
@@ -77,48 +80,50 @@ define([
     GlobalNav.prototype.constructor = GlobalNav;
 
     /**
-     * @Object templates. Contains basic navigation templates.
+     * @returns Object templates. Contains basic navigation templates.
      * It uses lo-dash template function.
      */
-    GlobalNav.prototype.templates = {
-        catalogList: _.template([
-            '<ul class="<%= classes.catalogList %>">',
-                '<img src="/source/assets/i/process.gif" alt="<%= labels.loading %>"/>',
-            '</ul>'
-        ].join("")),
-
-        catalogHeader: _.template('<h2 class="<%= classes.catalogListTitle %>"> <%= catalogMeta.title %></h2>'),
-
-        catalogMeta: _.template('<div class="<%= classes.catalogText %>"><%= catalogMeta.info %></div>'),
-
-        catalogLinkToAll: _.template([
-            '<li class="<%= classes.catalogListItem %> <%= classes.catalogListAll %>">',
-                '<a class="<%= classes.catalogLinkToAll %>" href="<%= url %>"><%= labels.linkToAllSpecs %> <%= length %> </a>',
-            '</li>'
-        ].join("")),
-
-        navigationListItem: _.template([
-            '<li class="<%= classes.catalogListItem %>">',
-                '<a class="<%= classes.catalogListLink %> source_a_g" href="#">',
-                    '<span class="<%= classes.catalogListTitle %>"></span>',
-                    '<div class="<%= classes.catalogListDate %>"></div>',
-                '</a>',
-            '</li>'
-        ].join("")),
-
-        catalogFilter: _.template('<div class="<%= classes.catalogFilter %>"></div>'),
-
-        togglePreviewLink: _.template('<button class="<%= classes.catalogImageThumbler %>"><%= togglePreviewLabel %></button>'),
-        updateButton: _.template('<button class="<%= classes.updateButton %>"><%= labels.updateButton %></button>'),
-
-        sortList: _.template([
-            '<ul class="source_sort-list">',
-                '<li class="source_sort-list_li">Sort by&nbsp;</li>',
-                '<li class="source_sort-list_li"><a class="source_sort-list_a" id="sortByAlph" href="#sort=alph">alphabet</a></li>',
-                '<li class="source_sort-list_li">&nbsp;or&nbsp;</li>',
-                '<li class="source_sort-list_li"><a class="source_sort-list_a" id="sortByDate" href="#sort=date">date</a></li>',
-            '</ul>'
-        ].join(""))
+    GlobalNav.prototype.initTemplates = function() {
```

Comment:
Please add a short section in navigation docs about styling and changing templates.